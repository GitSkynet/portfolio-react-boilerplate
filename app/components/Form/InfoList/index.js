/**
 *
 * InfoList
 *
 */

import styled from 'styled-components';

const InfoList = styled.ul`
  list-style: none;
  padding: 0;
`;

export default InfoList;
