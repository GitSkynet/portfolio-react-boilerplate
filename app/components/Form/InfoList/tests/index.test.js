import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import 'jest-styled-components';

import InfoList from '../index';

describe('<InfoList />', () => {
  it('should render and match the snapshot', () => {
    const renderedComponent = shallow(<InfoList />);
    expect(toJson(renderedComponent)).toMatchSnapshot();
  });
});
