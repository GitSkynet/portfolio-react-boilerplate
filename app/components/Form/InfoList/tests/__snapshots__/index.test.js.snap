// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InfoList /> should render and match the snapshot 1`] = `
.c0 {
  list-style: none;
  padding: 0;
}

<ul
  className="c0"
/>
`;
